README.md

```diff
@@ -287,6 +288,28 @@ DATABASE_URL="clickhouse://username:password@127.0.0.1:9000/database_name?on_clu
 
 [See other supported connection options](https://github.com/ClickHouse/clickhouse-go#dsn).
 
+#### BigQuery
+Follow the following format for `DATABASE_URL` when connecting to actual BigQuery in GCP:
+
+```
+bigquery://projectid/location/dataset?disable_auth=true
+```
```

Comment:
When connecting to actual BigQuery in GCP, `disable_auth=true` will never work, will it?

And, we are going to correctly express the project ID and dataset name in the URL path, so it should be:

```
bigquery:///projectid/dataset
```

Reply:
It will: https://github.com/go-gorm/bigquery/blob/master/driver/driver.go#L45-L47
Now that we are using the underlying client from the driver and not creating a new client within dbmate

Reply:
You are right. It's not able to do that. I will make the necessary changes for it.